int64 time_stamp_1
float32 roll
float32 pitch
float32 yaw   
float64 longitude
float64 latitude
#float32 height
float32 X
float32 Y 
#int32 time_stamp_2
int32 wheel_angle
int32 velocity
float32 velocity_x
float32 velocity_y

